osquery/core/database/rocksdb_migration.h

```diff
@@ -0,0 +1,33 @@
+/**
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <osquery/expected.h>
+#include <rocksdb/db.h>
+
+#include <unordered_map>
```

Comment:
Why do you need include of `<rocksdb/db.h>` and `<unordered_map>` here? I can't see any usage in the header file.